maintenance/pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>org.apache.camel.springboot</groupId>
         <artifactId>spring-boot-parent</artifactId>
-        <version>4.4.0-SNAPSHOT</version>
+        <version>4.8.0-SNAPSHOT</version>
```

Comment:
does it make sense to use 4.8.3-SNAPSHOT or is it enough to keep just the minor 4.8.0-SNAPSHOT and the build process will handle it?